stylesheets/pages/quiz.css

```diff
@@ -0,0 +1,53 @@
+@import url(../base.css);
+/* Quiz Page  */
+
+.quiz-question-wrapper{
+    width: 90%;
+    max-width: 500px;
+    border-radius: var(--space-xs);
+    gap: var(--space-xlg);
+    transition: background-color 0s;
+}
+
+.quiz-question-wrapper >*{
+    border-radius: var(--space-xs) ;
+    background: var(--tr-background-lighter);
+}
+
+.ques-info .timer-wrapper{
+    position: absolute;
+    left: 50%;
+}
+
+.question-label{
+    border-radius: var(--space-xs);
+    top: -20px;
```

Comment:
You can use rem here instead of px.

Reply:
Thank you for pointing it out, I have fixed it in the latest commit